src/FieldWarning/Assets/Units/Resources/Armory.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I feel like with the armory being json we can make this much easier to parse. Instead of having regex look for strings (eg. "__") we can make this into proper json like so:

`
[
    {
        'category':"TNK",
        'type': "TNK",
        'name': 'FastTank'
    },

    {
        'category':"TNK",
        etc..
    }
]

`